sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AppConfigurationExporterIntegrationTest.java

```diff
@@ -11,78 +15,52 @@
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.api.trace.Tracer;
 import io.opentelemetry.context.Scope;
-import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 import reactor.core.publisher.Mono;
 
 import java.nio.charset.StandardCharsets;
 import java.util.Optional;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.atomic.AtomicBoolean;
 
 import static com.azure.core.util.tracing.Tracer.DISABLE_TRACING_KEY;
-import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-@Tag("integration")
 public class AppConfigurationExporterIntegrationTest extends AzureMonitorTraceExporterTestBase {
 
     @Test
     public void setConfigurationTest() throws InterruptedException {
         CountDownLatch appConfigCountDown = new CountDownLatch(1);
-        CountDownLatch exporterCountDown = new CountDownLatch(2);
+        CountDownLatch exporterCountDown = new CountDownLatch(1);
 
+        Tracer tracer = configureAzureMonitorExporter(new ValidationPolicy(exporterCountDown, "AppConfig.setKey"));
         ConfigurationClient client = getConfigurationClient(appConfigCountDown);
-        Tracer tracer = configureAzureMonitorExporter((context, next) -> {
-            Mono<String> asyncString = FluxUtil.collectBytesInByteBufferStream(context.getHttpRequest().getBody())
-                .map(bytes -> new String(bytes, StandardCharsets.UTF_8));
-            asyncString.subscribe(value -> {
-                if (value.contains("app-config-exporter-testing") && value.contains("\"responseCode\":\"200\"")) {
-                    exporterCountDown.countDown();
-                }
-                if (value.contains("AppConfig.setKey")) {
-                    exporterCountDown.countDown();
-                }
-            });
-            return next.process();
-        });
 
-        Span span = tracer.spanBuilder("app-config-exporter-testing").startSpan();
+        Span span = tracer.spanBuilder("set-config-exporter-testing").startSpan();
         final Scope scope = span.makeCurrent();
         try {
             // Thread bound (sync) calls will automatically pick up the parent span and you don't need to pass it explicitly.
-            client.setConfigurationSetting("hello", "text", "World");
+            ConfigurationSetting configurationSetting = client.setConfigurationSetting("hello", "text", "World");
         } finally {
             span.end();
             scope.close();
         }
-        assertTrue(appConfigCountDown.await(1, TimeUnit.SECONDS));
-        assertTrue(exporterCountDown.await(1, TimeUnit.SECONDS));
+        assertTrue(appConfigCountDown.await(60, TimeUnit.SECONDS));
+        assertTrue(exporterCountDown.await(60, TimeUnit.SECONDS));
     }
 
+    @Disabled
```

Comment:
is this test disabled intentional?

Reply:
Yes, I have an issue to track this and re-enable it.